File: java/src/main/java/com/mertsy/javasample/ClipboardUtil.java
package com.mertsy.javasample;

import android.content.ClipData;
import android.content.ClipboardManager;
import android.content.Context;

public class ClipboardUtil {

    private final Context context;

    public ClipboardUtil(Context context) {
        this.context = context;
    }

    public void copy(String message) {
        ClipboardManager clipboard = (ClipboardManager) context.getSystemService(Context.CLIPBOARD_SERVICE);
        ClipData clip = ClipData.newPlainText("modelId", message);
        clipboard.setPrimaryClip(clip);
    }

}

File: kotlin/src/main/java/com/mertsy/kotlinsample/ClipboardUtil.kt
package com.mertsy.kotlinsample

import android.content.ClipData
import android.content.ClipboardManager
import android.content.Context

class ClipboardUtil(private val context: Context) {
    fun copy(message: String) {
        val clipboard = context.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
        val clip = ClipData.newPlainText("modelId", message)
        clipboard.setPrimaryClip(clip)
    }
}
